components/script_bindings/codegen/codegen.py

```diff
@@ -5772,7 +5778,7 @@ def declare(self, cgClass: CGClass) -> str:
             f"{returnType}{const}{override}{body}{self.breakAfterSelf}"
         )
 
-    def define(self, cgClass) -> None:
+    def define(self, cgClass: CGClass) -> None:
```

Comment:
```suggestion
    def define(self, cgClass: CGClass) -> str:
```
and suppress the error with `# pyrefly: ignore # bad-return` and change the `pass` to `assert False`.